src/commands/prepare.ts

```diff
@@ -527,7 +511,7 @@ export async function prepareMain(argv: PrepareOptions): Promise<any> {
 
   if (argv.publish) {
     logger.success(`Release branch "${branchName}" has been pushed.`);
-    await execPublish(newVersion);
+    await execPublish(argv.remote, newVersion);
```

Comment:
Why does `argv.remote` already exist here? Is it guaranteed to be defined and non-null?

Reply:
>Is it guaranteed to be defined and non-null?

This is TypeScript, what do you think? 🙂 

https://github.com/getsentry/craft/pull/245/files#diff-233373e3f179f999c4d8c851082724070fc138206b6254f2d080d299254a53b1R74-R78